src/charts/CytoViz/CytoViz.js

```diff
@@ -226,8 +376,176 @@ export const CytoViz = (props) => {
           </Tooltip>
         </div>
         <div className={classes.drawerContent}>
-          <TabPanel data-cy="cytoviz-drawer-details-tab-content" value={currentDrawerTab} index={0}>
-            {currentElementDetails || labels_.noSelectedElement}
+          <TabPanel
+            data-cy="cytoviz-drawer-details-tab-content"
+            value={currentDrawerTab}
+            index={0}
+            className={classes.tabPanel}
+          >
+            <Accordion
+              square
+              expanded={expandedPanel === 'nodeDetailsPanel'}
+              onChange={expandAccordeonPanel('nodeDetailsPanel')}
+            >
+              <AccordionSummary
+                aria-controls="nodeDetailsPanel-content"
+                id="nodeDetailsPanel-header"
+                expandIcon={<ExpandMoreIcon />}
+              >
+                {labels_.accordeon.nodeDetails}
+              </AccordionSummary>
+              <AccordionDetails>{currentElementDetails || labels_.noSelectedElement}</AccordionDetails>
+            </Accordion>
+            <Accordion
+              square
+              expanded={expandedPanel === 'findNodePanel'}
+              onChange={expandAccordeonPanel('findNodePanel')}
+            >
+              <AccordionSummary
+                aria-controls="findNodePanel-content"
+                id="findNodePanel-header"
+                expandIcon={<ExpandMoreIcon />}
+              >
+                {labels_.accordeon.findNode.headline}
+              </AccordionSummary>
+              <AccordionDetails>
+                <div className={classes.querySearchByID}>
+                  {labels_.accordeon.findNode.searchByID}
+                  <Autocomplete
+                    onChange={(event, node) => {
+                      if (node) {
+                        node.parent().data('hidden', false);
+                        node.closedNeighborhood().data('hidden', false);
+                        cytoRef.current?.animate({
+                          center: { eles: node },
+                          duration: 1000,
+                        });
+                        node.select();
+                      }
+                    }}
+                    options={nodesInCyto}
+                    getOptionLabel={(node) => node.data('label')}
+                    getOptionSelected={(option, node) => node.data('label') === option.data('label')}
+                    renderInput={(params) => <TextField {...params} variant="outlined" size="Small" />}
```

Comment:
Not sure about this but I think `Small` should be `small`, because we have warnings in the browser console:
```
Failed prop type: Invalid prop `size` of value `Small` supplied to `ForwardRef(FormControl)`, expected one of ["medium","small"].
```